Commands/Public/imgur.js

```diff
@@ -0,0 +1,120 @@
+const imgur = require("imgur");
+const { IsURL } = require("../../Modules/Utils/index");
+const { tokens: { imgurClientID } } = require("../../Configurations/auth");
+
+imgur.setClientId(imgurClientID);
+
+module.exports = async ({ Constants: { Colors, Text } }, documents, msg, commandData) => {
+	if (msg.suffix === "credits" && configJSON.maintainers.includes(msg.author.id)) {
+		const c = await imgur.getCredits();
+		return msg.send({
+			embed: {
+				color: Colors.INFO,
+				title: "Imgur Credits Info",
+				description: require("util").inspect(c.data)
```

Comment:
nit: At least surround it in a json codeblock

Reply:
It's 5 lines of name: value, it looks nice the way it is, a code block would just make it ugly. I even format it there to not look like a stringified object.